docs/multicluster/architecture.md

```diff
@@ -0,0 +1,87 @@
+# Antrea Multi-cluster Architecture
+
+Antrea Multi-cluster controller focuses on service discovery of containers across
+multiple clusters. Its implementation leverages Kubernetes and Kubernetes native
+solutions as much as possible.
+
+Antrea Multi-cluster controller depends on Antrea as its Kubernetes CNI to do Service
+and Endpoints traffic forwarding.
+
+Below diagram depicts a basic multi-cluster topology in Antrea.
+
+<img src="assets/basic-topology.png" width="400" alt="Antrea Multi-cluster Topology">
+
+Given a set of K8s clusters, there will be a leader cluster and several member clusters.
+By default, a leader cluster itself is also a member cluster of a multi-cluster set.
```

Comment:
Should we use the MCS term "ClusterSet"?

Reply:
done